packages/x-date-pickers/src/internals/hooks/useDesktopPicker/useDesktopPicker.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { resolveComponentProps, useSlotProps } from '@mui/base/utils';
 import MuiInputAdornment from '@mui/material/InputAdornment';
 import IconButton from '@mui/material/IconButton';
-import useForkRef from '@mui/utils/useForkRef';
+import { unstable_useId as useId, unstable_useForkRef as useForkRef } from '@mui/utils';
```

Comment:
I know that we are not following this rule well, but I would prefer to avoid root import from any package
We could add a lint rule at some point

See https://github.com/mui/material-ui/issues/35840

Reply:
We could do a once over the x packages and replace existing root imports. 🤔

Reply:
We would have to check if `@mui/utils` have clean nested export for everything (I think it does not have any `@mui/` deps re-exports so it should be fine)